pyshelf/search/container.py

```diff
@@ -0,0 +1,37 @@
+from elasticsearch_dsl.connections import connections
+from pyshelf.search.update_manager import UpdateManager
+from pyshelf.search.manager import Manager as SearchManager
+
+
+class Container(object):
+    def __init__(self, config, logger):
+        self.logger = logger
+        self.config = config
+        self._hosts = None
+        self._update_manager = None
+        self._search_manager = None
+        self.default_elastic_connection()
+
+    @property
+    def hosts(self):
+        if not self._hosts:
+            self._hosts = self.config.get("elasticSearchHost")
+
+        return self._hosts
+
+    def default_elastic_connection(self):
+        connections.create_connection("default", hosts=self.hosts)
```

Comment:
As discussed, instead we will pass in a "connection_string" which will get handed to the manager and the update manager to make connections.

In this way we won't be setting up global configuration and hopefully it will be easier to use and understand as well (even though it will take more coding time)